<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="check_remember_last_loaded_profile">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remember last loaded profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_remember_last_com_port">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remember last com port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_auto_connect_after_restart">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Try to connect automatically after program restart</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_beep_when_done">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Beep when done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_flash_when_done">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Flash color when done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_start_compact">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start with compact view</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>39</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
